common/spring-cloud-stream-modules-common-configuration/src/main/java/org/springframework/cloud/stream/module/trigger/TriggerProperties.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 the original author or authors.
+ * Copyright 2015-16 the original author or authors.
```

Comment:
Something wrong here with years...
Right, new/affected class should be updated to the 2016 already :smile: 
Minor, of course, and I can fix on merge, but need do not forget that, especially when IDE keeps Copyright collapsed...